modules/openapi-generator/pom.xml

```diff
@@ -433,6 +433,11 @@
             <artifactId>caffeine</artifactId>
             <version>2.8.1</version>
         </dependency>
+        <dependency>
+            <groupId>nl.jworks.markdown_to_asciidoc</groupId>
+            <artifactId>markdown_to_asciidoc</artifactId>
+            <version>1.1</version>
```

Comment:
While I'm sure this is a great tool, we should try to find a dependency that's been updated in the last 3 years. Including this will invariably introduce problems in consuming applications (e.g. plugins with requirements on newer versions of Kotlin, or any application which embeds openapi-generator and requires jsoup).